src/core/public/doc_links/doc_links_service.ts

```diff
@@ -133,7 +133,8 @@ export class DocLinksService {
           remoteClustersProxy: `${ELASTICSEARCH_DOCS}modules-remote-clusters.html#proxy-mode`,
           remoteClusersProxySettings: `${ELASTICSEARCH_DOCS}modules-remote-clusters.html#remote-cluster-proxy-settings`,
           scriptParameters: `${ELASTICSEARCH_DOCS}modules-scripting-using.html#prefer-params`,
-          transportSettings: `${ELASTICSEARCH_DOCS}modules-network.html`,
+          transportSettings: `${ELASTICSEARCH_DOCS}modules-transport.html`,
+          typesRemoval: `${ELASTICSEARCH_DOCS}removal-of-types.html`,
```

Comment:
The modules-transport.html page has been moved to modules-network.html as of 7.12. The removal-of-types.html page only exists as a redirect page in 8.0. They'll have to look at the 7.last docs.

Reply:
👍 thanks for the heads up. The transport settings url wasn't introduced in this PR, so I think it's best to address separately. 

The types removal doc is being used specifically for the 7--> 8 upgrade (when user is using UA on 7.last) so I think it is OK.